src/Illuminate/Cache/TaggedCache.php

```diff
@@ -104,7 +110,7 @@ public function taggedItemKey($key)
     /**
      * Fire an event for this cache instance.
      *
-     * @param  \Illuminate\Cache\Events\CacheEvent  $event
+     * @param  \Illuminate\Cache\Events\TaggedCacheEvent  $event
```

Comment:
This breaks it, remember that all events should now implement TaggedCacheEvent
TaggedCache extends Repository and overwrites `event`, for example `CacheHit`